### ISSUE TITLE
[Ios]Getting registrationId on version 1.5.1 but not on the lastest version.

### ISSUE THREAD

**Issue by guerrarj** [phonegap/phonegap-plugin-push#818]
### Expected Behaviour
Get registrationId on Ios using the latest version of the plugin.
For you guys having issues getting the registrationId on iOS, give the old version 1.5.1 a try and check if it works out for you.
### Actual Behaviour
Able to get the registrationId using the version 1.5.1 of the plugin but can´t get one using the latest version.
### Reproduce Scenario (including but not limited to)
  Add plugin name="phonegap-plugin-push" source="npm" on config.xml using phonegap Build service.

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
All iOS versions.

#### Plugin version
Everything works using version 1.5.1 with the exception of the 2 times notification on some iOS versions.

Can´t get registrationId using the latest version of the plugin.

**Comment by macdonst** [comment]
@guerrarj what version of cordova-ios are you using?

**Comment by guerrarj** [comment]
@macdonst Cordova - cli-5.2.0 and  (Could reproduce this on iOS 9.3 /  iOS 7.1.2 )

I am using the Phonegap Build Service as below:
PhoneGap (iOS / Android / Windows)
cli-5.2.0 (3.9.1 / 4.1.1 / 3.8.1)

Thank you for helping me out.

**Comment by macdonst** [comment]
@guerrarj right, the latest version of this plugin 1.6.2 requires cordova-ios 4.x or greater. You are using 3.9.1.

**Comment by guerrarj** [comment]
@macdonst OK. I have tried now using the version of build phonegap ( cli-6.0.0 ( cordova-ios 4.0.1 ) ) 
but the iOS 7.1.2 does not support it. Any ideas/workarounds on that situation ? Thanks.

**Comment by macdonst** [comment]
@guerrarj sorry if you absolutely need iOS 7 support you will be stuck with the earlier version of this plugin. Does upgrading to cordova-ios 4.x solve your original problem?

**Comment by guerrarj** [comment]
@macdonst OK. I will test it on the iOS 9.3 later today because I am not with the device here with me right now. I will put the results here later.

Thanks again.

**Comment by guerrarj** [comment]
@macdonst I was able to install and get the registrationId on iOS 9.3 using cordova-ios 4.0.1 but did not recieve notifications.

At the end I was able to fix the notification fired twice on iOS 9.3 by changing the payload and putting the "content_available" in the right place. (using the 1.5.1 version of the plugin)

So for me, my final solution was using the Plugin at version(1.5.1) , ios-cordova(3.9.1), phonegap(5.2.0)

This way I could install on iOS 7.1.2, iOS 9.3, Android and get registrationId and recieve notifications on all devices.

If you or anybody need some more details I can help.

Thank you.
